src/object_storage/objects.rs

```diff
@@ -109,6 +112,13 @@ impl Object {
         &self.c_name
     }
 
+    /// Object url
+    #[inline]
+    pub fn url(&self) -> Result<Url> {
```

Comment:
Could you move it below `name`? The last time I checked, the documentation showed items in the order they're defined.

Reply:
No problem.

I've done it in 7becce68f9d587e731ebb8df795d552f1f244cea.